<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManage</class>
 <widget class="QWidget" name="UserManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QTableView" name="role_view">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="delBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="sqlspace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>481</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:solid;
border-width:1px;
border-color:rgb(190,190,190);
background-color: rgb(255, 255, 255);
border-radius: 6px; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbt_Refresh">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>41</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Agency FB&quot;;
color: rgb(255, 255, 255);
background-color: rgb(78, 160, 201);
border-radius: 5px; 
</string>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
